x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -412,7 +410,7 @@ public void updateApiKeys(
             return;
         }
 
-        final TransportVersion version = getMinNodeTransportVersion();
+        final Version version = getMinNodeVersion();
```

Comment:
are we bringing the Version back here? how will this work in serverless?

Reply:
At the moment I am primarily thinking about getting rid of `transportVersion`. I noticed that the uses of `getMinNodeTransportVersion` weren't actually about serialisation, and were just checking against version constants. These can remain as `Version` for now.

At some point the security team will need to work out what to do with this, but for the moment, this particular instance can stay as `Version`.